cli/flox-rust-sdk/src/models/environment/mod.rs

```diff
@@ -228,15 +229,19 @@ impl PathPointer {
 pub struct ManagedPointer {
     pub owner: EnvironmentOwner,
     pub name: EnvironmentName,
+    pub floxhub_web_url: Url,
+    pub floxhub_git_url: Url,
```

Comment:
suggestion:  @ghudgins has said this is okay as is, but I don't think we should serialize both of these as users will see them. And there isn't currently any time when we can't just prepend `git.`. If we need the git url to be different for testing, just use the env var, don't expose a second configuration option to every user

Reply:
It was pretty annoying writing unit tests for this when the git host isn't provided.
I agree in general that for _actual_ floxhub instances we can likely either prepend .git or ping an api that has all the urls (see e.g. GitHub's api).
OTOH this file is user _visible_ but not supposed to be _edited_ manually anyway..

But I can look at how we can remove the git url as well if that's blocking

Reply:
I think it would be worth having a single config before introducing multiple configurations.

Reply:
1. The file we are talking about here is not a configuration file
2. If you can only set floxhub globally that's almost contradictory to being able to set an alternative. I.e you will be locked out of _all_ other hubs including the default.

This file is meant to make environments portable by including the upstream in the pointer you commit to your project

Reply:
I would expect the pointer to be portable and take precedence. But we would have a single base url from which we could derive any additional urls we need

Reply:
yes, that was my goal too!

however, that base url then becomes a hard requirement which basically defies testing with `file:///`  as well as a local `floxhub` instance running under an IP address or `localhost` as there wont be a hostname to derinve anything from.

I dont think requiring a local instance of floxhub with resolv rules and just to test core git interaction is reaonable

Reply:
Ok it optional now and only being set if a git url override is applied